tests/backup/backup_test.go

```diff
@@ -482,31 +493,34 @@ var _ = Describe("{BasicBackupCreation}", func() {
 			monthlyPolicyStatus := Inst().Backup.BackupSchedulePolicy(monthlyName, uuid.New(), orgID, monthlySchedulePolicyInfo)
 			dash.VerifyFatal(monthlyPolicyStatus, nil, "Creating monthly schedule policy")
 		})
-		Step("Register cluster for backup", func() {
+		Step("Registering cluster for backup", func() {
+			log.InfoD("Registering cluster for backup")
 			ctx, err := backup.GetAdminCtxFromSecret()
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
 			dash.VerifyFatal(err, nil, "Creating source and destination cluster")
 			clusterStatus, clusterUid = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
-			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, "Verifying backup cluster")
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying backup cluster with uid [%s]", clusterUid))
 		})
-		Step("Taking backup of applications", func() {
+		Step("Taking backup of all namespaces", func() {
+			log.InfoD("Taking backup of all namespaces")
 			ctx, err := backup.GetAdminCtxFromSecret()
 			dash.VerifyFatal(err, nil, "Getting context")
-			for _, namespace := range bkpNamespaces {
-				backupName = fmt.Sprintf("%s-%s-%v", BackupNamePrefix, namespace, time.Now().Unix())
+			for index, namespace := range bkpNamespaces {
+				backupName = fmt.Sprintf("%s-%s-%v", BackupNamePrefix, namespace, currentTimestamp+int64(index*2))
```

Comment:
Let us not add int to timestamp. Either wait for 2 seconds or just use random strings as suffix.
Store the backup names in an array and use this array for cleanup.

Reply:
Used random strings as suffix and stored the backup and restore names in an array to facilitate the deletion of backup and restore operations.